exporter/alibabacloudlogserviceexporter/README.md

```diff
@@ -0,0 +1,26 @@
+# AlibabaCloud LogService Exporter
+
+This exporter supports sending OpenTelemetry data to [LogService](https://www.alibabacloud.com/product/log-service)
+
+Configuration options:
+
+- `endpoint` (required): LogService's [Endpoint](https://www.alibabacloud.com/help/doc-detail/29008.htm).
+- `project` (required): LogService's Project Name.
+- `logstore` (required): LogService's Logstore Name.
+- `access_key_id` (optional): AlibabaCloud access key id.
+- `access_key_secret` (optional): AlibabaCloud access key secret.
+- `ecs_ram_role` (optional): set AlibabaCLoud ECS ram role if you are using ACK.
+- `max_retry` (optional): max retry count when send fail.
+- `max_buffer_size` (optional): max buffer used in memory.
```

Comment:
Sending data to LogService with the producer is best practice, the producer will use different retry strategy according to the error code from the server. The max retry count and max buffer size are very important for the producer, so we expose these params.

Reply:
@tigrannajaryan what are your thoughts? Is it a showstopper if retries and buffers are not being managed by the processors? I assume so given it defeats the purpose of having those things centrally managed but just verifying.

Reply:
@jrcamp To avoid conceptual conflicts, I have removed the producer and use a normal client instead. And we will use some processors to do retry/batch/memory limit. If we have more requirements for processors, I will submit some processor PRs.

Reply:
@jrcamp  Thanks for reminding us, we can use queuedprocessor directly. Actually we will use the combination of memorylimiter&batchprocessor&queuedprocessor.